### PR TITLE
ravindra _Assignment_11

### PR DESCRIPTION
**Pull request by RavindraMule**
Assignment_11
image

### CODE REVIEW

File: src/ravindra/MaxMinNumber.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//Assignment11
+package ravindra;
+
+public class MaxMinNumber {
+	void minNumber(int num1,int num2) {
+		if(num1<num2) {
```
Comment (AnvitP) [inline src/ravindra/MaxMinNumber.java:6]: If you pass the values17,10 it wont print anything.... please update the condition and make changes accordingly

**Comment by RavindraMule** [comment]
changes added
image

File: src/ravindra/MaxMinNumber.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			System.out.println(num1+" is smaller between "+num1 +" and "+num2+".");
+		}
+		else {
+			System.out.println(num2+" is bigger between "+num1 +" and "+num2+".");
```
Comment (AnvitP) [inline src/ravindra/MaxMinNumber.java:10]: Here in print statement instead of bigger, it should be smaller

**Comment by RavindraMule** [comment]
changes added
image

**Comment by AnvitP** [comment]
Please check line 12, need to change that too

**Comment by RavindraMule** [comment]
changed added
image

**Pull request merged**